### ISSUE TITLE
Keyboard focus is not at all visible on "Install", "search" and "Playground" control: A11y_TypeSpec_Keyboard

### ISSUE THREAD

**Issue by kupatkar99** [microsoft/typespec#8556]
### GIT Hub Tags:
#A11yTCS; #A11ySev2; #TypeSpec; #GH_TypeSpec_Web_Sept2025; #DesktopWeb; #A11yMAS; #WCAG2.4.7; #FocusVisible; #AILimited; #Win11; #FTP; #ChromiumEdge; #Keyboard;

### Environment and OS details:
Application name: typespec.io
URL: https://[typespec.io](https://typespec.io/)/
ChromiumEdge Version: 140.0.3485.66 (Official build) (64-bit)

### Reproduction Steps:

1. Open url  https://[typespec.io](https://typespec.io/)/ and login with valid Credentials.
2. "TypeSpec" Page will get open.
3. TAB to "Search button.
4. Observe the issue whether Keyboard Focus is clearly visible on "Install", "search" and "Playground" control or not.

### Actual behavior:
 Keyboard focus is not at all visible on "Install", "search" and "Playground" controls.

### Expected behavior:
Keyboard Focus should be clearly visible on "Install", "search" and "Playground" Controls.

### User Impact:
Users who rely on keyboard navigation are unable to determine which control is currently active, making it difficult or even impossible for them to interact with these controls independently.

### Attachment:
https://github.com/user-attachments/assets/9107896a-863b-4917-a831-cd76fa3b7197

**Comment by ccbarragan** [comment]
Suggested focus order and labels:

<img width="1598" height="777" alt="Image" src="https://github.com/user-attachments/assets/b7f0e6f5-bb95-48fd-ac17-81c2f5462a56" />

https://www.figma.com/design/8iipnzLVistMDEC1sg6Evh/TypeSpec-website?node-id=1728-57055&t=DJyE9s6LOtvFi4lm-1

**Comment by kupatkar99** [comment]
@timotheeguerin Verified the issue in below environment:
https://[typespec.io](https://typespec.io/)/
issue is partially fixed. Focus is not visible on "Install" and "Playground" buttons. Hence Reactivating the bug. please Find added attachment for reference.

https://github.com/user-attachments/assets/edbad077-300c-488b-975b-cb683e2b80e4
